examples/01_encodings.py

```diff
@@ -62,11 +62,11 @@
 # set of columns: some are numerical (``'year_first_hired'``), some dates
```

Comment:
some are dates ?

Reply:
Actually, I think that this is proper English